corenlp/src/main/scala/org/clulab/processors/corenlp/CoreNLPProcessor.scala

```diff
@@ -195,6 +205,7 @@ class CoreNLPProcessor(
     if(doc.sentences.head.syntacticTree.isEmpty)
       throw new RuntimeException("ERROR: you have to run the constituent parser before discourse parsing!")
 
+    // TODO Becky: uncomment these two lines after retraining
```

Comment:
?

Reply:
@MihaiSurdeanu they're uncommented -- did you want me to keep them, comment them, or remove them?  for what it's worth, I didn't see these lines, and so I didn't comment them out for retraining -- do I need to repeat the training? (sorry if you told me about this -- I have no memory about it)